pkg/hintrunner/zero/zerohint_others_test.go

```diff
@@ -4,11 +4,56 @@ import (
 	"testing"
 
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
 )
 
 func TestZeroHintMemcpy(t *testing.T) {
 
 	runHinterTests(t, map[string][]hintTestCase{
+		"MemcPyContinueCopying": {
+			{
+				operanders: []*hintOperander{
+					{Name: "continue_copying", Kind: uninitialized},
+				},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					ctx.ScopeManager.EnterScope(map[string]any{})
+					err := ctx.ScopeManager.AssignVariables(map[string]any{
```

Comment:
you can do this in one line, entering the scope with a variable inside it, this will allow to remove `AssignVariables` function call

Reply:
for sure!

Reply:
done! ✅